### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ashmalzahra**
This is a replica of [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=dhcLlOECv6H2MFvZ-0) desktop version, built using **media queries**. All elements are laid out using grid and flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,27 +56,25 @@ <h2>I am a Software Developer</h2>
...
-                        <li>html</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
-                    <button class="see-project" type="button">See Project</button>
-                </article>
-            </div>
-            <article class="project">
+            <h3 class="heading">My Recent Works</h3>
+            <hr id="line">
```
Comment (abdulhamiid) [inline index.html:60]: Your design :point_down: 
stud

Figma design :point_down: 
figmax

As stated in the project requirements it is important to replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please kindly adjust the right and left spacing to align the content in the center to match figma design (the line is too close to the right) please fix that.

The text color in the figma design is not black. let's fix that also (color: #091E42;)

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ashmalzahra :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

### Readme file :recycle: 
- [ ] Please let's consider removing the Micorverse template logo from the README file, as we are required to customize the readme according to our project and it is also part of the microverse rule that every student should remove the Microverse template logo in their project. 

- [ ] Optionally, you can replace it with your own logo if have any :+1: 
## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -138,28 +200,158 @@ div p {
...
+}
+
+@media (max-width: 767px) {
+  #line {
+    width: 327px;
+    margin-left: inherit;
+    margin-top: 0;
+    margin-bottom: 50px;
+  }
+}
```
Comment (BregornOriginal) [inline style.css:341]: - [ ] Kindly fix the line as the reviewer mentioned. It shouldn't go through the text.

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @ashmalzahra  

## Great work on making the changes requested by a previous reviewer 👏 
## You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

one-step-closer-one-more

 ## Highlights

✅ Linters are passing
✅ Good commit history
✅ Correct use of GitHub Flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @ashmalzahra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Nice UI design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**